### PR TITLE
Added a link and description to projects about the fccokc github repo…

### PR DESCRIPTION
**Pull request by stu7**
This pull request resolves issue #35

### CODE REVIEW

**Review by mattstub — CHANGES_REQUESTED** [review]
@stu7 I would propose that we remove the original wording about the website cloned, and also the new comments seem a little lengthy, perhaps we can remove the line about contacting Kimberly & Carson as the CONTRIBUTING file should help steer people that way.

(edit: for spelling mistakes)

**Comment by mattstub** [comment]
@stu7 how are these revisions coming? Everything working out okay?

**Comment by stu7** [comment]
@mstub Was out of town part of last week and busy this weekend. Will look at it soon and make the mods.

**Comment by mattstub** [comment]
@stu7 great! just wanted to make sure nothing was hanging you up to bad

**Comment by stu7** [comment]
@mstub Good week for work, bad week for coding. More of the same for Mon/Tues, working out of state and travelling this weekend. We'll see how much I can get done to finally finish this one up.

File: docs/projects.html [hunk 1]
```diff
@@ -79,10 +79,10 @@ <h3 id="fcc-okc-web">FreeCodeCamp Oklahoma City Website</h3>
...
+                        <strong>Project Lead:</strong><a href="https://github.com/dashcraft" target="_blank" rel="external"> Daniel Ashcraft</a> and <a href="https://github.com/CarsonMckinstry" target="_blank" rel="external">Carson McKinstry</a><br>
+                        <strong>Website:</strong> <a href="http://www.fccokc.com/" target="_blank" rel="external">http://www.fccokc.com/</a>
+                    </p>                    
+                    <h3 id="FCC Github">FCC OKC Github</h3><p class ="github repo">Free Code Camp OKC welcomes aspiring developers to become contributors to the OKC website. There are a number of issues we are seeking to address across our website and they are posted to our <a href="https://github.com/FreeCodeCampOKC/fccokc_web" target="_blank" title="FCC OKC Github repo" rel="external">Github</a>. If you would like to contribute to this opensource endeavour contact FCC OKC about your interest to become a website contributor.</p>
```
Comment (mattstub) [inline docs/projects.html]: Could you move line 85 down to the bottom of the page? Under the twitch bot project? In its current place it feels like the page is broken into two sections.. thanks @stu7

**Review by mattstub — CHANGES_REQUESTED** [review]
Please note the 1 change about line 85. Everything else looks good. Thanks!

**Review by mattstub — APPROVED** [review]

**Pull request merged**